src/service/poc_lora.proto

```diff
@@ -80,9 +80,9 @@ message lora_witness_report_req_v1 {
   // Timestamp of witness received in nanos since unix epoch
   uint64 timestamp = 4;
   uint32 tmst = 5;
-  // Signal in ddbm
+  // Signal in ddBm
   sint32 signal = 6;
-  // SNR in ddbm
+  // SNR in ddB
```

Comment:
To avoid any confusion, I'm dropping the "m" here because SNR is a pure dimensionless quantity. It's not even relative to the milliwatt.